packages/ui-overlays/src/Mask/props.ts

```diff
@@ -32,12 +33,15 @@ import type {
 import type { WithStyleProps, ComponentStyle } from '@instructure/emotion'
 
 type MaskOwnProps = {
+  children?: React.ReactNode
   onDismiss?: (...args: any[]) => any
   placement?: 'top' | 'center' | 'bottom' | 'stretch'
   fullscreen?: boolean
   onClick?: (...args: any[]) => any
```

Comment:
These event handlers can be typed better

Reply:
Yeah, I'm sorry, I haven't finished this ticket yet but accidentally made a PR for it already (reflex I guess). There are stuff I want to marked as deprecated and I wanted to wait for https://github.com/instructure/instructure-ui/pull/856 to be merged before I do it. Sorry about the confusion and thanks for the review anyway :)

Reply:
Done! Had to deprecate `onDismiss`, it hasn't been used in the component for a long time.